### PR TITLE
Integrate `0.41.0-adhoc.x0a2709a2` platform sdk

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Integrate `0.41.0-adhoc.x0a2709a2` platform sdk

### CODE REVIEW

**Review by edward-swirldslabs — APPROVED** [review]

**Comment by JeffreyDallas** [comment]
Why another one when there is a pending one 

https://github.com/hashgraph/hedera-services/pull/7601

**Comment by Neeharika-Sompalli** [comment]
> Why another one when there is a pending one
> 
> #7601

I think we can close that

**Review by JeffreyDallas — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM, thx @Neeharika-Sompalli !

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/ProtectedFilesUpdateSuite.java [hunk 1]
```diff
@@ -191,9 +191,7 @@ private HapiSpecOperation[] validateAndCleanUpOps(
             getAccountBalance(account).hasTinyBars(changeFromSnapshot("preUpdate", 0))
         };
         HapiSpecOperation[] opsArray = {
-            validateOp,
-            UtilVerbs.updateLargeFile(account, fileName, fileName),
-            getFileContents(fileName).hasContents(fileName)
```
Comment (jsync-swirlds) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/ProtectedFilesUpdateSuite.java:196]: lost a line here?

**Review by jeromy-cannon — DISMISSED** [review]

**Review by edward-swirldslabs — APPROVED** [review]
Signing off on the files owned by platform-hashgraph

File: platform-sdk/sdk/settings.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ sync.syncProtocolPermitCount,             2
                                           #############################
 
 freezeSecondsAfterStartup,                0
-event.creation.maxCreationRate,           5
+event.creation.maxCreationRate,           20
```
Comment (edward-swirldslabs) [inline platform-sdk/sdk/settings.txt:24]: Was this change intentional?  If so, might want to have Austin or Cody weigh in on its value.

**Review by edward-swirldslabs — APPROVED** [review]

File: platform-sdk/sdk/settings.txt (hunk 2 above)
Comment (Neeharika-Sompalli) [reply platform-sdk/sdk/settings.txt]: Not intentional. Thanks @edward-swirldslabs

File: hedera-node/configuration/mainnet/settings.txt [hunk 3]
```diff
@@ -30,6 +30,7 @@ event.enableEventStreaming,                    true
 event.eventsLogDir,                            /opt/hgcapp/eventsStreams
 event.eventsLogPeriod,                         5
 event.maxEventQueueForCons,                    1000
+event.creation.maxCreationRate,                1
```
Comment (alittley) [inline hedera-node/configuration/mainnet/settings.txt:33]: The `maxCreationRate` for mainnet shouldn't be `1`.

IMO, you should only change the value for whichever config file manages the integration tests in the PR.

It still remains to be seen what a reasonable value will be for mainnet and other networks

Comment (Neeharika-Sompalli) [reply hedera-node/configuration/mainnet/settings.txt]: Thanks @alittley . Removed from mainnet and testnet settings

**Review by jeromy-cannon — APPROVED** [review]

**Review by alittley — APPROVED** [review]

File: hedera-node/configuration/compose/settings.txt [hunk 4]
```diff
@@ -30,6 +30,7 @@ event.enableEventStreaming,                    false                        # di
 event.eventsLogDir,                            /opt/hgcapp/eventsStreams
 event.eventsLogPeriod,                         5
 event.maxEventQueueForCons,                    1000
+event.creation.maxCreationRate,                1
```
[the remark this answers is not in the document]
Comment (Neeharika-Sompalli) [reply hedera-node/configuration/compose/settings.txt:33]: I think compose is used for some other tests, not very sure. I can confirm and remove.

**Review by povolev15 — APPROVED** [review]

**Review by alittley — DISMISSED** [review]

**Review by alittley — APPROVED** [review]

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/ProtectedFilesUpdateSuite.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/ProtectedFilesUpdateSuite.java:196]: Yes, this was intentional. Had to fix a test.

**Review by povolev15 — APPROVED** [review]
